### ISSUE TITLE
Getting an error when trying to run apt-get upgrade

### ISSUE THREAD

**Issue by matthieu-ravey** [phusion/passenger-docker#92]
Here is the beginning of my Dockerfile:

```
FROM phusion/passenger-ruby21:0.9.14

RUN apt-get -y update
RUN apt-get -y upgrade
```

And I get the following error when trying to build the image:

```
Errors were encountered while processing:
 nginx-common
 nginx-extras
E: Sub-process /usr/bin/dpkg returned an error code (1)
INFO[0064] The command [/bin/sh -c apt-get -y upgrade] returned a non-zero code: 100 
```

**Comment by gmeans** [comment]
I get the same issue. I think its related to the conf scripts for nginx and redis. Here's output from earlier in the upgrade:

```
Installing new version of config file /etc/nginx/koi-utf ...

Configuration file '/etc/nginx/nginx.conf'
 ==> Modified (by you or by a script) since installation.
 ==> Package distributor has shipped an updated version.
   What would you like to do about it ?  Your options are:
    Y or I  : install the package maintainer's version
    N or O  : keep your currently-installed version
      D     : show the differences between the versions
      Z     : start a shell to examine the situation
 The default action is to keep your current version.
*** nginx.conf (Y/I/N/O/D/Z) [default=N] ? dpkg: error processing package nginx-common (--configure):
 EOF on stdin at conffile prompt
dpkg: dependency problems prevent configuration of nginx-extras:
 nginx-extras depends on nginx-common (= 1:1.6.3-8.5.0.8~trusty1); however:
  Package nginx-common is not configured yet.

dpkg: error processing package nginx-extras (--configure):
 dependency problems - leaving unconfigured
Setting up libruby2.2:amd64 (2.2.2-1bbox1~trusty1) ...
Setting up ruby2.2-dev:amd64 (2.2.2-1bbox1~trusty1) ...
Setting up ruby2.2 (2.2.2-1bbox1~trusty1) ...
Setting up libruby2.1:amd64 (2.1.6-1bbox1~trusty1) ...
Setting up ruby2.1-dev:amd64 (2.1.6-1bbox1~trusty1) ...
Setting up ruby2.1 (2.1.6-1bbox1~trusty1) ...
Setting up passenger-doc (1:5.0.8-1~trusty1) ...
Setting up unattended-upgrades (0.82.1ubuntu2.2) ...
Setting up python3-software-properties (0.92.37.3) ...
Setting up software-properties-common (0.92.37.3) ...
Setting up redis-server (2:3.0.1-rwky1~trusty) ...

Configuration file '/etc/redis/redis.conf'
 ==> Modified (by you or by a script) since installation.
 ==> Package distributor has shipped an updated version.
   What would you like to do about it ?  Your options are:
    Y or I  : install the package maintainer's version
    N or O  : keep your currently-installed version
      D     : show the differences between the versions
      Z     : start a shell to examine the situation
 The default action is to keep your current version.
*** redis.conf (Y/I/N/O/D/Z) [default=N] ? dpkg: error processing package redis-server (--configure):
 EOF on stdin at conffile prompt
```

**Comment by gmeans** [comment]
Scratch that, this is fixed using the solution mentioned in #83.

Add this to the dockerfile instead:

```
RUN apt-get update && apt-get install -y -o Dpkg::Options::="--force-confold" passenger nginx-extras
```

Updated to only be passenger, not passenger-enterprise.

**Comment by FooBarWidget** [comment]
I've added explicit upgrade instructions to the documentation:
https://github.com/phusion/passenger-docker#upgrading_os
https://github.com/phusion/passenger-docker#upgrading_passenger
